.dockerignore

```diff
@@ -0,0 +1,8 @@
+.env
+.env.template
+env_vars
+.gitignore
+Vagrantfile
+.vscode
+geckodriver
```

Comment:
You still have the geckodriver file in the repo.

Remove it from the repo without actually deleting the file locally by running `git rm --cached geckodriver`